src/docs/patterns/forms.mdx

```diff
@@ -82,3 +95,78 @@ In some specific situations, you may need to defer verification or perform it fo
         '2: Use a circular progress indicator when waiting for responses from the server.',
     ]}
 />
+
+## Submit the Form
+
+Usually, a form will have a submit/next/finished button at the end. This button is typically placed at the bottom right corner of the form. If you are designing for a right-to-left language, this button should be at the bottom left.
+
+<ImageGrid
+    images={[SumbitButtonLocationImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={[
+        '1: On desktops, the button is placed in the bottom corner of the form.',
+        '2: On mobile devices, the button is occluded by the native keyboard.',
+    ]}
+/>
+<ImageGrid
+    images={[NextButtonUsageImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={[
+        'On mobile, if there are more than one input fields on the screen, the keyboard button should say "next".',
+        '1: The user entered a valid input. Clicking "next" will autofocus the user to the next text field.',
+        '2: The user entered a invalid input. Clicking "next" will display the error message but will not autofocus to the next text field.'
+    ]}
+/>
+
+### Disable the Submit Button
+
+When there are not a lot of text fields, the submit button can be disabled to indicate that there are some errors in the user input.
+
+<ImageGrid
+    images={[SubmitButtonOnChangeImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={
+        'If you have only one input field, and this field is verified on change, disable the submit button until the user entered the desired input.'
+    }
+/>
+<ImageGrid
+    images={[SubmitButtonAutoSubmitImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={
+        'For the verification code screen, you can auto-submit the code once the user input reaches the desired length (2). After the verification code is recognized (3), the screen pauses for a short moment before auto-transit to the next page. The verify button is preserved for consistency.'
+    }
+/>
+
+If at least one text field uses the 'verify on blur' strategy, do not disable the submit button. Otherwise, the mobile user will have to disable their keyboard first to have their input field verified, before they can click the submit button. The only exception is if there are required input fields.
```

Comment:
This is only relevant on mobile right? Desktop users shouldn't be disabling their keyboards 😛 

If above we are saying that onBlur should be the most common strategy, this scenario will be a very common one.  What is the proper way of doing this?

Should we mention somewhere about being able to submit the form with the Enter key (for desktop users)?

Reply:
Good idea for the enter key :) 

On blur for most scenarios; thus do not disable the submit button in most cases. 

Eh, I should just have a bullet list of do-disable-submit-button scenarios (instead of the other way)